Packs/CortexXDR/Scripts/XDRIncidentWidgetBase/XDRIncidentWidgetBase.py

```diff
@@ -0,0 +1,60 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    try:
+        res_type = demisto.getArg('reultType')
+        query_type = demisto.getArg('queryType')
+        to_date = demisto.getArg('to')
+        from_date = demisto.getArg('from')
+        list_name = f'xdrIncidents_{query_type}'
+
+        # parse time arguments
+        date_pattern = re.compile('\d{4}[-/]\d{2}[-/]\d{2}T\d{2}:\d{2}:\d{2}')
+
+        if to_date and to_date != '0001-01-01T00:00:00Z':
+            result = date_pattern.findall(to_date)[0]
+            to_date = datetime.strptime(result, '%Y-%m-%dT%H:%M:%S')
```

Comment:
Why not use the `strptime` with the `to_date` variable?

Reply:
Because `to_date` could be in different format, depends on the date range value